cromwell_tools/cromwell_api.py

```diff
@@ -127,13 +127,13 @@ def health(cls, auth, raise_for_status=False):
 
     @classmethod
     @retry(reraise=True, wait=wait_exponential(multiplier=1, max=10), stop=stop_after_delay(20))
-    def submit(
-            cls, auth, wdl_file, inputs_file, options_file=None, inputs_file2=None, zip_file=None,
-            collection_name=None, label=None, validate_labels=True, on_hold=False):
-        # TODO: Allow more inputs files rather 2 to be consistent with the Cromwell API
-        # TODO: Add `raise_for_status` flag to this method and remove the hard-coded `raise_for_status()` call
-        # TODO: Purifying the methods by taking the specific retry policy out from this method
-        """ Submits a workflow to Cromwell
+    def submit(cls, auth, wdl_file, inputs_file, options_file=None, inputs_file2=None, zip_file=None,
+               collection_name=None, label=None, validate_labels=False, on_hold=False):
```

Comment:
The default for `validate_labels` is currently set to `True`: https://github.com/broadinstitute/cromwell-tools/blob/master/cromwell_tools/cromwell_tools.py#L140. Why change it here?

Reply:
Sounds good to me! We should update this so that it does not throw any errors on the newer versions of Cromwell